pack/ur.json

```diff
@@ -53,6 +53,24 @@
         "type_code": "hardware",
         "uniqueness": true
     },
+    {
+        "code": "26069",
+        "cost": 0,
+        "deck_limit": 3,
+        "faction_code": "anarch",
+        "faction_cost": 1,
+        "flavor": "Win a new friend today!",
+        "illustrator": "Elizaveta Sokolova",
+        "keywords": "Chip",
+        "pack_code": "ur",
+        "position": 69,
+        "quantity": 3,
+        "side_code": "runner",
+        "text": "[trash]: Set aside the top 6 cards of your stack. You may install 1 program or <strong>virtual</stron> resource from among the set aside cards, paying 2{credit] less. Shuffle 3 of the remaining cards into your stack, then remove the rest from the game.",
```

Comment:
closing tag should be `</stronG>`. Opening bracket in `{credit]` should be square